cmd/operator/test/e2e/e2e_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
-	"github.com/kubarchive/kubearchive/test/utils"
+	"github.com/kubearchive/kubearchive/operator/test/utils"
```

Comment:
I think this needs to be updated

```suggestion
	"github.com/kubearchive/kubearchive/cmd/operator/test/utils"
```